pages/design/rule-template.md

```diff
@@ -61,12 +61,40 @@ _There are no major accessibility support issues known for this rule._
 
 ## Background
 
-- Links to Techniques for WCAG 2.0
-- Latest version: Techniques for WCAG 2.0 W3C Working Group Note 8 April 2014
-- Link to other methodologies, test specifications or tools
-- Links to test cases, test suites, unit tests, etc.
-- The WCAG 2.0 Techniques already contain examples and code snippets to illustrate which content passes or fails the test. Whenever possible WCAG-ACT-RULES-CG refers to those. Another source for test cases is the W3C Before and After Demonstration.
-- Other references
+<!-- Add an introduction if you want to add further information not available on another page -->
+
+### Understanding WCAG 
+
+<!-- Add links to understanding documents for all success criteria the rule maps to, either directly or through a composite rule. Remove if not applicable -->
+
+- [WCAG 2.0 - Understanding 4.1.2: Name, Role, Value](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html)
```

Comment:
I suggest changing the link to the WCAG 2.1 version: [WCAG 2.1 - Understanding 4.1.2: Name, Role Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

I know it is only an example, but we might as well teach people to go look for the 2.1 resources.